client/components/Admin/TopPage.tsx

```diff
@@ -47,8 +47,7 @@ const TopPage: FC<{}> = () => {
           {/* TODO: multiple nodes */}
           {searchConfigured ? (
             <>
-              Configured: {host}
-              {indexName}, <strong>{esVersion}</strong>
+              Configured: {host}/{indexName}, <strong>{esVersion}</strong>
```

Comment:
Maybe `host` or `indexName` is including `/` already.

Reply:
No, actually not.

It parsed and separated to host and indexName
https://github.com/crowi/crowi/blob/e455d11ec5b29a17a7ee8b6d8fabe54a4f023c30/lib/util/search.ts#L125

Reply:
Hm.. in case of no index name specified?

Reply:
Oh, that's it!
In my environment, `ELASTICSEARCH_URL` is `elasticsearch:9200/`.

Reply:
Okay... I see.
Anyway, both host and indexName spec should be fixed even any env passed.

Reply:
I confirmed `{host}` not includes `indexName`.

![image](https://user-images.githubusercontent.com/29064/64104334-0518e800-cdaf-11e9-89e8-b232af410e55.png)

I added the test if the index name not specified and set `crowi` as default.
https://github.com/crowi/crowi/pull/557/commits/b97b12c21ed2ca3f688565dd131d0525b0a759d2#diff-a51ffab4bb0b2ada968f2932a43d1497R23

Reply:
Ah, I think the cause is different from this.
The true cause is that passed `uri` is not matched with `/^(https?:\/\/[^/]+)\/(.+)$/`.
So matched string is assigned to `host` and `indexName` and these remains at the initial values.

Reply:
Because Crowi uses the ES URL to determine index name with parsing it.
So the protocol should be specified.
(In addition, the es client adds the defaultProtocol if it is no specified https://github.com/elastic/elasticsearch-js-legacy/blob/16.x/src/lib/host.js#L86 )

So I changed below:
- Error when the protocol not specified.
- And use [WHATWG API](https://nodejs.org/docs/latest/api/url.html#url_whatwg_api) to parse it.

Reply:
Thanks for your description. Okay, it's LGTM.